src/ScalarToTecplot.h

```diff
@@ -3,6 +3,7 @@
 
 #include "Scalar.h"
 #include <string>
+#include <string.h>
```

Comment:
Why does this need to be included?

Reply:
On several machines, the compiler has thrown the following error:

src/ScalarToTecplot.cc:116:51: error: 'strncpy' was not declared in this scope

Including <string.h> fixed this error. Including <cstring> will also fix this error. It seems that both <string.h> and <cstring> contain functions for C-style strings, but <string.h> is deprecated in C++ (https://stackoverflow.com/questions/9257665/difference-between-string-and-string-h). The ScalarToTecplot function must be trying to use a function for C-style strings, so it seems that either <string.h> or <cstring> must be included. 

Can you check if the code compiles on your machine, or if you get the same error I did? If you do get the same error, I'll change the include statement to <cstring>, otherwise I'll remove the include statement.

Reply:
PR #10 fixes this issue.